.github/workflows/tests.yaml

```diff
@@ -0,0 +1,331 @@
+# WIP github actions
+name: Tests
+
+# Controls when the action will run.
+on:
+  pull_request:
+    branches: [ master ]
```

Comment:
If you use `$default-branch` here it won't break when we eventually switch to `main`

Reply:
Good catch.